docs/guides/getting-started/bpmn/gettingstarted_quickstart_advanced.bpmn

```diff
@@ -1,13 +1,12 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0o87biy" targetNamespace="http://bpmn.io/schema/bpmn" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="1.0.0">
+<?xml version="1.0" encoding="UTF-8"?><bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0o87biy" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Web Modeler" exporterVersion="b72d66c" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="1.0.0" camunda:diagramRelationId="565f5fc0-a7f4-43ad-9fe5-0a1213312401" xmlns:camunda="http://camunda.org/schema/1.0/bpmn">
   <bpmn:process id="camunda-cloud-quick-start-advanced" name="Camunda Cloud Quick start" isExecutable="true">
```

Comment:
@Hafflgav since you are updating this file, WDYT about updating the process id and name? I deliberately didn't do this ahead of the C8 launch because I didn't touch any asset files.

Reply:
How could I have missed your comment. Seems like I was drowning in GitHub Notifications... 
Great idea. I will do so and push a new commit today.